tools/driver/autolink_extract_main.cpp

```diff
@@ -145,6 +150,39 @@ extractLinkerFlagsFromObjectFile(const llvm::object::ObjectFile *ObjectFile,
   return false;
 }
 
+static bool
+extractLinkerFlagsFromBitcodeFile(const llvm::object::IRObjectFile *ObjectFile,
+                                  std::vector<std::string> &LinkerFlags,
+                                  CompilerInstance &Instance) {
+  // Search for the section we hold autolink entries in
+  for (auto &M : ObjectFile->modules()) {
+    StringRef FlagsString;
+    for (auto &GV : M.globals()) {
+      if (!GV.hasSection() || !GV.hasInitializer())
+        continue;
+      auto *EntriesConstant = dyn_cast<llvm::ConstantDataArray>(GV.getInitializer());
+      if (GV.getSection() == ".swift1_autolink_entries" &&
```

Comment:
This should only be done if the module is ELFish.

Reply:
I changed to skip if the module is not elfish.